quadrat/block-templates/index.html

```diff
@@ -0,0 +1 @@
+<!-- wp:template-part {"slug":"index"} /-->
```

Comment:
We have an `index.html` in the block-template-parts folder than the index.php references — should we use that instead?